src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -401,6 +401,25 @@ public void NamespaceWithoutName()
             Assert.Equal(string.Empty, symbol.Name);
         }
 
+        [WorkItem(539740, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/539740")]
+        [Fact]
+        public void SingleLineNamespaceWithoutName()
+        {
+            var text = "namespace;";
+            var tree = Parse(text);
+            var comp = CreateCompilation(tree);
+            var model = comp.GetSemanticModel(tree);
+            var errors = comp.GetDiagnostics().ToArray();
+            Assert.Equal(2, errors.Length);
```

Comment:
nit: Let's check the diagnostics instead of just the count

Reply:
I am OK with this because it's taken from the test directly above.